roles/django_app_docker/tasks/freeports.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Copy free ports script to host
+  copy:
+    content: |
+        #!/usr/bin/python3
+        import socket
+        for p in range({{ django_app_docker_port_range.start }},{{ django_app_docker_port_range.end }}):
+            import socket
+            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            try:
+              s.bind(("",p))
+            except:
+                pass
+            port = s.getsockname()[1]
+            s.close()
+            if port != 0:
+              print(p)
+    dest: /tmp/freeports.py
+    mode: 777
```

Comment:
777 is way too open, please only assign permissions that are actually needed. 700 seems much more suitable, since it's sudo user copying the script anyway.

Reply:
updated to 700.